apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/Inflector.kt

```diff
@@ -72,9 +74,8 @@ private fun irregular(): List<Pair<String, String>> {
       "move" to "moves")
 }
 
-private fun uncountable(): List<String> {
-  return listOf("equipment", "information", "rice", "money", "species", "series", "fish", "sheep")
-}
+private val uncountable = listOf("equipment", "information", "rice", "money", "species", "series", "fish", "sheep")
```

Comment:
missed this earlier, what is this for?

Reply:
To proper support plural / singular wording.